components/search/SearchResultsGrid.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <b-card-group
+    id="searchResults"
+    deck
+  >
+    <ContentCard
+      v-for="result in results"
+      :key="result.europeanaId"
+      :name="result.fields.dcTitle[0]"
+      :url="localePath({ name: 'record-all', params: { pathMatch: result.europeanaId.slice(1) } })"
+      :image-url="result.edmPreview"
+      :texts="cardTexts(result)"
+      data-qa="search result"
+    />
+  </b-card-group>
+</template>
+
+<script>
+  import ContentCard from '../generic/ContentCard';
+
+  export default {
+    components: {
+      ContentCard
+    },
+    props: {
+      results: {
+        type: Array,
+        default: () => []
+      }
+    },
+    methods: {
+      cardTexts: function (result) {
+        let texts = [];
+        if (Array.isArray(result.fields.dcCreator)) {
+          let creator =  result.fields.dcCreator.slice(0, 3).join(this.$t('formatting.listSeperator') +' ');
```

Comment:
These should probably be marked up as lists in the HTML, with styling achieved via CSS.

Reply:
Let's do that later.

Reply:
Agreed.